qiskit/circuit/quantumcircuit.py

```diff
@@ -4683,6 +4684,199 @@ def mcp(
             copy=False,
         )
 
+    def mcrx(
```

Comment:
It would be nice to polish these docs a bit, see e.g. the docstring for `mcp` above, which looks a bit nicer in the docs 🙂 Should the type of the qubits also be `QubitSpecifier` here?